db/db_metrics.c

```diff
@@ -510,11 +510,10 @@ static time_t metrics_standing_queue_time(void) {
 }
 
 static void update_standing_queue_time(void) {
-    double qdepth = time_metric_average(thedb->queue_depth) + time_metric_average(thedb->handle_buf_queue_time);
```

Comment:
`queue_depth` measures the size of request queues (sqlenginepool and handle_buf). `handle_buf_queue_time` measures time (in milliseconds) waiting in handle_buf's queue. They're 2 different things, hence it doesn't make a lot sense to add them together. Moreover, using `handle_buf_queue_time` seems to favor spikes too much: One 100-millisecond queuing event followed by 98 non-queuing events would still be considered queuing (`100/99 > 1`).

The patch takes `handle_buf_queue_time` out.